lib/Wikia/src/CircuitBreaker/ExternalCircuitBreakerStorage.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
Do we need to keep the external cicruit breaker at all?

Reply:
Let's leave it as an example. I've removed references to it